app/src/components/mode-specific/desktop/InterceptTraffic/WebTraffic/TrafficExporter/HarImportModal/index.tsx

```diff
@@ -0,0 +1,138 @@
+import { Col, Modal, Row } from "antd";
+import SpinnerColumn from "components/misc/SpinnerColumn";
+import React, { useCallback, useState } from "react";
+import { useDropzone } from "react-dropzone";
+import { FiUpload } from "react-icons/fi";
+// import { convertHarJsonToRQLogs } from "../harLogs/converter";
+import { Har } from "../harLogs/types";
+import SessionSaveModal from "views/features/sessions/SessionsIndexPageContainer/NetworkSessions/SessionSaveModal";
+import {
+  trackHarImportButtonClicked,
+  trackHarImportCanceled,
+  trackHarImportCompleted,
+} from "modules/analytics/events/features/sessionRecording/networkSessions";
+import { RQButton } from "lib/design-system/components";
+
+interface Props {
+  onSaved: (sessionId: string) => void;
+  btnText?: string;
+}
+
+const ImportandSaveNetworkHarModalButton: React.FC<Props> = ({ onSaved, btnText }) => {
```

Comment:
Too long name. Also, not consistent with the package name, as it is a default export. Can we rename both package and the component to `HarImportModalButton`? Also, the import references can be updated.